source/_elephpants/2020-09-06-white-symfonycasts.md

```diff
@@ -0,0 +1,19 @@
+---
+categories:
+    - symfonycasts
+tags:
+    - white
+sponsor: SymfonyCasts
+reverse: SymfonyCasts Logo
+photos:
+    -
+        file: sfcasts-white-0-400x300.jpg
+        link: https://www.flickr.com/photos/leannapelham/52745291949/
+        credit: Leanna Pelham
+    -
+        file: sfcasts-white-1-400x300.jpg
+        link: https://www.flickr.com/photos/leannapelham/52745037496/
+        credit: Leanna Pelham
+---
+When it comes to watching PHP screencasts, this elephant is a real couch potato. Created with
+a Symfonycasts community member, this elephant had a very small run and remains elusive in the wild.
```

Comment:
There are 5, right?

Reply:
Yep! Currently there are only 5 out in the world